pod/locale/fr/LC_MESSAGES/django.po

```diff
@@ -3058,8 +3056,8 @@ msgid ""
 "a>."
 msgstr ""
```

Comment:
I'm not fan of strings begin with empty "" (eg msgid "" or msgstr "") but as they are so much, this seems to be the rule !

Reply:
It's automatically generated when executing `make lang`.

So, it's normal :)